### ISSUE TITLE
Format config file not found.

### ISSUE THREAD

**Issue by janwilmans** [Caphyon/clang-power-tools#836]
clang-format accepts both .clang-format and _clang-format files on windows, however, for my projects where I have _clang-format, clang power tools reports:

```
--- ERROR ---
Format config file not found.
Create the config file and place it in the solution folder or select one of the predefined format styles from Clang Power Tools settings -> Format -> Style.
```

**Comment by Enache-Ionut** [comment]
Hi @janwilmans,

Thank you for taking the time to report this issue which is now fixed and it will be available in the next Clang Power Tools version, v6.1. 

If you have any other problems or feature suggestions let us know.

Regards,
~Ionut

**Comment by janwilmans** [comment]
thanks!
